src/modules/accommodations/accommodations.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { Controller, Get, Query } from '@nestjs/common';
+import { AccommodationsService } from './accommodations.service';
+import { SearchAccommodationParamsDto } from './dto/request';
+import { GetAccommodationsSwaggerDecorator } from './decorators/swagger.decorator';
+
+@Controller('accommodations')
+export class AccommodationsController {
+  constructor(private readonly accommodationsService: AccommodationsService) {}
+
+  @Get()
+  @GetAccommodationsSwaggerDecorator()
+  async getManyAccommodations(
+    @Query() searchParams: SearchAccommodationParamsDto,
```

Comment:
You are calling it ...ParamsDto but using Query decorator. Be consistent in naming